lib/tests/structure.js

```diff
@@ -138,6 +139,19 @@ function declareTests(options = { logLevel: 'warn' }) {
                                 fs.realpathSync(path.join(schemaDir, `${latestSchemaInfo.version}.${options.contentTypes[0]}`)));
                         });
                     }
+
+                    if (options.enforcedNumericBounds) {
+                        it(`Should not have minimum or maximum values outside the bounds specified in options ${latestSchemaInfo.path}`, () => {
+                            const enforcedMin = options.enforcedNumericBounds[0];
+                            const enforcedMax = options.enforcedNumericBounds[1];
+                            traverseSchema(latestSchemaInfo.schema, (obj) => {
+                                if (['number', 'integer'].includes(obj.type)) {
+                                    assert.ok(obj.minimum >= enforcedMin, 'has a minimum value lower than allowed by the config');
```

Comment:
Do we need to check that obj.minimum is set before comparing it?  It might not be if something has gone wrong, or someone has manually edited a materialized file.

Reply:
Yep you're right, didn't think of the possibility of manipulating the materialized schema. Adding that check.